### ISSUE TITLE
HL crash if clipboard has non text data

### ISSUE THREAD

**Issue by Eiyeron** [openfl/lime#1289]
Hello, I'm trying a few things here and there with lime's hashlink integration.

Using the develop branch (cac8250), I can trigger a 100% crash when screenshotting in Windows, even on default projects. Here's the traceback
```
Uncaught exception: Access violation
Called from $String.fromUTF8(E:\Dev\Haxe\HaxeToolkit\haxe\std/hl/_std/String.hx:224)
Called from lime.system.$Clipboard.__update(lime/system/Clipboard.hx:39)
Called from lime._internal.backend.native.NativeApplication.handleClipboardEvent(lime/_internal/backend/native/NativeApplication.hx:209)
Called from lime.app.Application.exec(lime/app/Application.hx:167)
Called from $ApplicationMain.start(ApplicationMain.hx:137)
Called from $ApplicationMain.create(ApplicationMain.hx:127)
Called from $ApplicationMain.main(ApplicationMain.hx:18)
Called from fun$5499(?:1)
```

I have a wild guess : lime seems to get the clipboard's event and data but can't process it (as it's not text but image data?). I haven't delved in much more though, sorry.

Have a nice day.

**Comment by jgranick** [comment]
String handled in HL is something we still need to investigate... who owns the pointer? How best to convert from a Haxe string to be used on the native side, and vice-versa?

Or, perhaps this is a null pointer, where the clipboard has no data

**Comment by player-03** [comment]
Seems to be the exact same issue as #1344. And if 1344 was closed by 0ab93e033724bf2148e1af3a15ab849267b187b3, this should be too.
